main.py

```diff
@@ -0,0 +1,253 @@
+import pygame
+import sys
+
+clock=pygame.time.Clock()
+
+from pygame.locals import *
+
+import random
+
+pygame.mixer.pre_init(44100, -16, 2, 2048)
+pygame.init()
+pygame.mixer.set_num_channels(64)
+
+pygame.display.set_caption('    World Runner')
+
+WINDOW_SIZE= (600,400)
+
+screen =pygame.display.set_mode(WINDOW_SIZE)
+
+display = pygame.Surface((300,200))
+
+player_image = (pygame.image.load('player_animations/idle/idle1.jpg').convert_alpha())
+
+grass_image = pygame.image.load('map_files/grass.jpg').convert_alpha()
+grass_image= pygame.transform.scale(grass_image,(16,16))
+TILE_SIZE = grass_image.get_width()
+lava_image=pygame.image.load('map_files/lava.png').convert_alpha()
+lava_image= pygame.transform.scale(lava_image,(16,16))
+plant_image=pygame.image.load('map_files/plant.png').convert_alpha()
+plant_image= pygame.transform.scale(plant_image,(16,16))
+
+tile_index={1:grass_image,2:lava_image,3:plant_image}
+
+chunk_size=8
+
+def generate_chunk(x,y):
```

Comment:
every function should have docstrings explaining inputs, outputs, and goal of the function.